vos/vos/commands/vchmod.py

```diff
@@ -110,6 +112,6 @@ def vchmod():
         logger.error("Received keyboard interrupt. Execution aborted...\n")
         sys.exit(getattr(ke, 'errno', -1))
     except Exception as e:
-        logger.error("Error: " + str(e))
+        logger.error(e.msg)
```

Comment:
Its not clear to me that all Exception classes have the error message in their .msg attribute. 

mach147:~ kavelaarsj$ cat test.py 
try:
    raise BaseException("An Exception")
except BaseException as ex:
   print str(ex)
   print ex.msg
   pass

results in: 

An Exception
Traceback (most recent call last):
  File "test.py", line 5, in <module>
    print ex.msg
AttributeError: 'exceptions.BaseException' object has no attribute 'msg'